### PR TITLE
data:Crazycoder122

### PR DESCRIPTION
**Pull request by Crazycoder122**

### CODE REVIEW

**Review by senali-d — COMMENTED** [review]
Look Good

**Review by dan-mba — APPROVED** [review]
Looks good to me

File: data/Crazycoder122.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+   "name":"Krishnendu Dasgupta",
+   "displayStatsPublic":true,
+   "type":"personal",
+   "bio":"Programmer by passion and also a Musician. Trying to solve the toughest Algorithms of life through Music..",
+   "avatar":"https://github.com/Crazycoder122.png",
+   "links":[
+      {
+         "name":"Krishnendu Dasgupta",
```
Comment (kumarsonsoff3) [inline data/Crazycoder122.json:9]: I recommend updating link name to something like, `Twitter`, or `Follow me on Twitter`.

File: data/Crazycoder122.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+         "url":"https://twitter.com/KrishnenduDG",
+         "icon":"FaTwitter"
+      },
+      {
+         "name":"Krishnendu Dasgupta",
```
Comment (kumarsonsoff3) [inline data/Crazycoder122.json:14]: What do you think on updating it to something like -  `LinkedIn`, or `Connect with me on LinkedIn`, etc?

File: data/Crazycoder122.json (hunk 1 above)
Comment (eddiejaoude) [reply data/Crazycoder122.json:9]: Good suggestions 👍 I will merge this PR and improvements to the profile can come in another PR

**Pull request merged**